docs/reference/field-types/oembed.md

```diff
@@ -0,0 +1,57 @@
+# `oembed`
+
+A `oembed` field supports the user in embedding media hosted by any [oembed—compatible hosting site](https://oembed.com/#section7), or any site for which you have provided an [oembetter](https://github.com/apostrophecms/oembetter) filter via the `@apostrophecms/oembed` module. <!-- TODO: document oembetter config for 3.x -->
+
+The field will immediately preview the media embed after entering a valid URL.
+
+The database value of the field will have `url`, `title` and `thumbnail` properties. `title` and `thumbnail` are snapshots from the oembed response at the time the field was saved. `thumbnail` is the URL of a thumbnail image as provided by the oembed response. Developers should retrieve the full embed code in client-side code to get the latest version available.
```

Comment:
We should probably mention the API call that can be used to do that, or link to `video.js` as an example.